<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_InsertPropertiesEditingWidget</class>
 <widget class="QWidget" name="LC_InsertPropertiesEditingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1244</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="buttonGroup8">
     <property name="title">
      <string>Geometry</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,5,0">
      <item row="0" column="0">
       <widget class="QLabel" name="lInsertionPointX">
        <property name="text">
         <string>Insertion point (x):</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>leInsertionPointX</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leInsertionPointX">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>X coordinate of insertion point</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="LC_PointPickButton" name="wPickInsertionPoint" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>46</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Pick insertion point coordinates from drawing</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lInsertionPointY">
        <property name="text">
         <string>Insertion point (y):</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>leInsertionPointY</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leInsertionPointY">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Y coordinate of insertion point</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lFactorX">
        <property name="text">
         <string>Scale X:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>leScaleX</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leScaleX">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Scaling factor by X axis</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="tbPIckScaleX">
        <property name="toolTip">
         <string>Pick scale by X value from drawing</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/icons/icons.qrc">
          <normaloff>:/icons/interactive_pick_distance.lci</normaloff>:/icons/interactive_pick_distance.lci</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lFactorY">
        <property name="text">
         <string>Scale Y:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>leScaleY</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="leScaleY">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Scaling factor by Y axis</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QToolButton" name="tbPIckScaleY">
        <property name="toolTip">
         <string>Pick scale Y value from drawing</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/icons/icons.qrc">
          <normaloff>:/icons/interactive_pick_distance.lci</normaloff>:/icons/interactive_pick_distance.lci</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lAngle">
        <property name="text">
         <string>Angle:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>leAngle</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="leAngle">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Rotation angle</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QToolButton" name="tbPickAngle">
        <property name="toolTip">
         <string>Pick Angle value from drawing</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/icons/icons.qrc">
          <normaloff>:/icons/interactive_pick_angle.lci</normaloff>:/icons/interactive_pick_angle.lci</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lRows">
        <property name="text">
         <string>Rows:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>leRows</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="leRows">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Amount of rows for copies</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lCols">
        <property name="text">
         <string>Columns:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>leCols</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="leCols">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Amount of columns for copies</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lRowSpacing">
        <property name="text">
         <string>Row Spacing:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>leRowSpacing</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="leRowSpacing">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Spacing between rows</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QToolButton" name="tbPickRowSpacing">
        <property name="toolTip">
         <string>Pick row spacing value from drawing</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/icons/icons.qrc">
          <normaloff>:/icons/interactive_pick_distance.lci</normaloff>:/icons/interactive_pick_distance.lci</iconset>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lColSpacing">
        <property name="text">
         <string>Column Spacing:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>leColSpacing</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="leColSpacing">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Spacing between columns</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QToolButton" name="tbPickColumnSpacing">
        <property name="toolTip">
         <string>Pick column spacing from drawing</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../../res/icons/icons.qrc">
          <normaloff>:/icons/interactive_pick_distance.lci</normaloff>:/icons/interactive_pick_distance.lci</iconset>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <spacer name="spacer58">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LC_PointPickButton</class>
   <extends>QWidget</extends>
   <header>lc_pointpickbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leInsertionPointX</tabstop>
  <tabstop>leInsertionPointY</tabstop>
  <tabstop>leScaleX</tabstop>
  <tabstop>tbPIckScaleX</tabstop>
  <tabstop>leScaleY</tabstop>
  <tabstop>tbPIckScaleY</tabstop>
  <tabstop>leAngle</tabstop>
  <tabstop>tbPickAngle</tabstop>
  <tabstop>leRows</tabstop>
  <tabstop>leCols</tabstop>
  <tabstop>leRowSpacing</tabstop>
  <tabstop>tbPickRowSpacing</tabstop>
  <tabstop>leColSpacing</tabstop>
  <tabstop>tbPickColumnSpacing</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../res/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
